src/editing.md

```diff
@@ -1,463 +1,489 @@
+Rechts davon schließen sich die Formatierungswerkzeuge an. Fett, Kursiv und Unterstrichen
+funktionieren, wie man es von anderen Textverarbeitungsprogrammen gewohnt ist.
+Mit den nächsten beiden Schaltflächen kannst du Text tief- und hochstellen, was
+praktisch sein kann für chemische Bindungen wie H<sub>2</sub>0 oder einfache
+mathematische Ausdrücke wie x<sup>2</sup>.
 
+Der Radiergummi entfernt die Formatierung des aktuell markierten Textes -
+einschließlich Farben, Schriftstärke und -stil, usw.
 
-### Using Fields
+Dahinter reihen sich drei Werkzeuge, die der Erstellung von Listen sowie
+Textausrichtung und -einrückung dienen, und zwei Schaltflächen, mit denen
+du die Schriftfarbe ändern kannst.
 
-For those who like to stay very organized, you can add fields to your
-notes to classify your content, such as "book", "page", and so on. Anki
-supports searching in specific fields, which means you can do a search
-for "book:'my book' page:63" and immediately find what you're looking
-for.
+Die Schaltfläche \[…\] ist nur sichtbar, wenn ein Lückentext-Notiztyp ausgewählt ist.
 
-### Custom Study and Filtered Decks
+Du kannst mit dem Büroklammer-Werkzeug Audiodateien, Bilder und Videos von
+deiner Festplatte auswählen und sie an deine Notiz anhängen.
+Alternativ kannst du Medien auch in die Zwischenablage deines Computers
+geben (z. B. durch einen Rechtsklick auf ein Bild im Internet und die Auswahl
+von "Bild kopieren") und in einem Feld deiner Wahl einfügen.
+Für nähere Informationen zu Medien, siehe das Kapitel [Medien](media.md).
 
-Using [custom study and filtered deck](filtered-decks.md) you can create 
-temporary decks out of search
-terms. This allows you to review your content mixed together in a single
-deck most of the time (for optimum memory), but also create temporary
-decks when you need to focus on particular material, such as before a
-test. The general rule is that if you always want to be able to study
-some content separately, it should be in a normal deck; if you only
-occasionally need to be able to study it separately (for a test, when
-under a backlog, etc.), then filtered decks created from tags, flags, 
-marks or fields are better.
+Mit einem Klick auf das Mikrofon kannst du eine Audiospur über das Mikrofon
+deines Computers aufnehmen und in das aktuelle Feld einfügen.
 
-## Editing Features
+Die Schaltfläche F<sub>x</sub> öffnet ein Menü, über welches du
+[MathJax oder LaTeX](math.md) in die Notiz einfügen kannst.
 
-The editor is shown when [adding notes](editing.md), [editing a
-note](studying.md) during reviews, or [browsing](browsing.md).
+Die Schaltfläche `</>` zeigt den zugrundeliegenden HTML-Code eines Feldes an.
 
-On the top left are two buttons, which open the [fields](editing.md#customizing-fields) and
-[cards](templates/intro.md) windows.
+Anki 2.1.45+ unterstützt das Anheften von Feldern direkt aus dem Editor.
+Wenn du auf die Stecknadel rechts oben in einem Feld klickst, wird Anki den
```

Comment:
This might be one of those rare differences in German vs. Austrian language. I didn't even know that word :smile:  But as Khabib would have put it: "Austria - 9 million, Germany - 80 million."